modules/virt-deploying-tto.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="virt-deploying-tto_{context}"]
+= Deploying the Tekton Tasks Operator resources
+
+The Tekton Tasks Operator (TTO) cluster tasks and example pipelines are not deployed by default when you install {VirtProductName}. To deploy TTO resources, enable the `deployTektonTaskResources` feature gate in the `HyperConverged` custom resource (CR).
+
+.Procedure
+
+. Open the `HyperConverged` CR in your default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Set the `spec.featureGates.deployTektonTaskResources` field to `true`.
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: kubevirt-hyperconverged
+spec:
+  tektonPipelinesNamespace: <userNamespace> <1>
```

Comment:
@geetikakay Can you please provide more information about that?

Reply:
@sjhala-ccs if we set tektonPipelinesNamespace : testnamespace ,(just an example) how will it impact pipelines creation. 
Maybe an example will help but since i have not tried this use case i am not sure what will be the output. I can surely try it out and see

Reply:
@ksimon1 Do you have any thoughts on this?

Reply:
the `tektonPipelinesNamespace` will not affect deployment of clusterTasks in any way. Only example pipelines will be deployed in specified namespace

Reply:
Thanks, @ksimon1. So, IIUC, I don't need to add or change anything here?